tests/test_functions.py

```diff
@@ -1101,8 +1102,8 @@ def run_outside_algorithm(
         self, ts, prior_distr="lognorm", standardize=False, ignore_oldest_root=False
     ):
         span_data = SpansBySamples(ts)
-        Ne = 0.5
-        priors = ConditionalCoalescentTimes(None, Ne, prior_distr)
+        Ne = np.array([[0, 0.5]])
```

Comment:
I'm not quite understanding why we need to give an array of 2 values here? Is this the timepoint and the Ne? I suspect we could just assume that if a single scalar was given, if could apply to all time.

Reply:
(also, I wonder if the parameterization should be `[array_of_times, array_of_Ne]` rather than `[[t0, Ne0], [t1, Ne1], ...]`?)

Reply:
It's just converting from single scalar to the "piecewise population size" format, which is epoch start (measuring time moving backwards) in the first column, and effective population size in the second column. The conversion can happen later, it'd probably be better to consolidate into the rescaling function itself.

I don't have strong opinions about whether the array should be 2-row or 2-column -- whatever you think is clearer. I do think it should be a 2d numpy array rather than a list of lists, so there's no ambiguity about dimensionality.

Reply:
I agree about consolidating into the rescaling function. See my other comments about a list of arrays (to allow different lengths for timebreaks and population sizes, meaning that we don't need to require a first "break" at time 0.